docker-hub/onboarding-faqs.md

```diff
@@ -111,3 +134,15 @@ organization. [Learn more](orgs.md#configure-repository-permissions).
 
 Docker SSO allows only one IdP configuration per organization. For more
 information, see [Configure SSO](../single-sign-on/index.md) and [SSO FAQs](../single-sign-on/faqs.md).
+
+### What is a service account?
+
+A [service account](../docker-hub/service-accounts.md) is a Docker ID used for automated management of container images or containerized applications. Service accounts are typically used in automated workflows, and do not share Docker IDs with the members in the Team or Business plan. Common use cases for service accounts include mirroring content on Docker Hub, or tying in image pulls from your CI/CD process.
+
+### Does my organization need to use Docker's registry?
```

Comment:
A registry is a hosted service containing repositories of images that responds to the Registry API. Docker Hub is Docker's primary registry, but you can use Docker with other container image registries. You can access The default registry by browsing [Docker Hub](https://hub.docker.com) or using the `docker search` command.